versioned_docs/version-2.0.0/running-keploy/configuration-file.md

```diff
@@ -44,65 +56,259 @@ test:
     containerName: ""
     networkName: ""
     # example: "test-set-1": ["test-1", "test-2", "test-3"]
-    tests:
+    selectedTests:
     # to use globalNoise, please follow the guide at the end of this file.
     globalNoise:
         global:
             body: {}
             header: {}
-        test-sets:
     delay: 5
+    buildDelay: 30s
     apiTimeout: 5
-    passThroughPorts: []
+    tests:
+        filters:
+            - path: ""
+              urlMethods: []
+              headers: {}
+              host: ""
+    stubs:
+        filters:
+            - path: ""
+              host: ""
+              port: 0
     withCoverage: false
     coverageReportPath: ""
 ```
 
-The presence of this file allows the user to eliminate the monotonous way of specifying the same parameters/flags for each record or test command in every run. The flags specified in the Keploy's [CLI Command Docs](http://keploy.io/docs/running-keploy/cli-commands/) are now converted into parameters to the record and test command in the config file. These parameters take the same input as the flags in the CLI command. Specifying those parameters in the config file will allow the user to directly run the record or test command without specifying the flags as shown below:
+## Using the Config File
+
+The Keploy-config file eliminates the need to repeatedly specify parameters for each record or test command. The parameters in the file correspond to the flags in the Keploy [CLI Command Docs](http://keploy.io/docs/running-keploy/cli-commands/). To run a record or test command, use:
```

Comment:
the last statement can be modified to :
"Using keploy-config can help reduce the record and test command to just:"